pandas/core/frame.py

```diff
@@ -7451,52 +7451,61 @@ def to_period(self, freq=None, axis=0, copy=True):
 
     def isin(self, values):
         """
-        Return boolean DataFrame showing whether each element in the
-        DataFrame is contained in values.
+        Whether each element in the DataFrame is contained in values.
 
         Parameters
         ----------
-        values : iterable, Series, DataFrame or dictionary
+        values : iterable, Series, DataFrame or dict
             The result will only be true at a location if all the
             labels match. If `values` is a Series, that's the index. If
-            `values` is a dictionary, the keys must be the column names,
+            `values` is a dict, the keys must be the column names,
             which must match. If `values` is a DataFrame,
             then both the index and column labels must match.
 
         Returns
         -------
+        DataFrame
+            DataFrame of boolean showing whether each element in the DataFrame
+            is contained in values.
 
-        DataFrame of booleans
+        See Also
+        --------
+        DataFrame.eq: Equality test for DataFrame.
+        Series.isin: Equivalent method on Series.
```

Comment:
Can you in addition also mention `Series.str.isin` ?

Reply:
When trying to look at `Series.str.isin`, I get this (latest HEAD on master):
```
>>> import pandas as pd
>>> help(pd.Series.str.isin)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'StringMethods' has no attribute 'isin'
```
Am I missing something?

Reply:
Sorry, my bad, I think I was getting confused with another method...

I think `Series.str.contains` would be a good addition to the `See Also`, as it's kind of the inverse.